eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -484,11 +484,22 @@
           "justification": "Removing Jackson annotations from Tables in transition to stream-style."
         },
         {
-          "regex": true,
           "code": "java\\.annotation\\.removed",
           "old": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "new": ".*? com\\.azure\\.communication\\.jobrouter\\.models.*",
           "justification": "Removing Jackson annotations from Job Router in transition to stream-style."
+        },
+        {
+          "regex": true,
+          "code": "java\\.annotation\\.attributeRemoved",
+          "old": ".*? com\\.azure\\.resourcemanager\\.network\\.models.*",
+          "justification": "Removing Jackson annotations from Network in transition to stream-style."
```

Comment:
wrong reason.

Just say "Jackson annotation changed" should be enough.

Reply:
Fixed in the new version.